daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -471,37 +491,44 @@ func (l *Libtelio) Refresh(c mesh.MachineMap) error {
 		return nil
 	}
 
-	meshnetMap := string(c.Raw)
+	var config teliogo.Config
+	if err := json.Unmarshal(c.Raw, &config); err != nil {
+		return fmt.Errorf("failed to unmarshall config: %d", err)
+	}
 
-	result := teliogo.TELIORESOK
+	var err error = nil
 	for i := 0; i < 10; i++ {
-		if result = l.lib.SetMeshnet(meshnetMap); result == teliogo.TELIORESOK {
+		if err := l.lib.SetMeshnet(config); err == nil {
```

Comment:
This will result in `nil` if error happens here. See: https://go.dev/play/p/FhGdo8N4P6T

Reply:
You got me! :smile: 
Fixing